src/middlewares/reportTo.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I don't really understand this report-to mechanism.

Edit: Ok, things make more sense after reading the route implementation, perhaps it would be good to mention the route, and what happens to these reports once they get sent there.

Reply:
I added a few more words about that.